packages/maskbook/src/extension/options-page/DashboardComponents/WalletContent.tsx

```diff
@@ -113,6 +124,9 @@ export const WalletContent = forwardRef<HTMLDivElement, WalletContentProps>(func
     }, [])
     //#endregion
 
+    //#collectibles loading
```

Comment:
Maybe move `useERC721TokenDetailed` to this component, life will be simpler.

Reply:
The best way is trigger suspense fallback loading..so we have a library to turn the async function to suspend? like 
 `getERC721TokenDetailed`

Reply:
Sounds great! Let's rewrite those utility hooks to suspends.

Reply:
ok

Reply:
Since `useERC721TokenDetailed` uses other hooks, it still need to be named `useERC721TokenDetailed`.